### DISCUSSION TITLE
Footprint charts?

### DISCUSSION THREAD

**Discussion by jinusean** [klinecharts/KLineChart#466] [Ideas]
Any chance footprint charts will be supported in the future?

**Comment by liihuu** [comment]
What is footprint?

**Reply by jinusean** [reply]
![image](https://github.com/liihuu/KLineChart/assets/6874981/5d2efbcb-cf5e-410f-98cc-7d6718b4e54c)
Left is market sells (hitting the bid), right is market buys (hitting the asks).

**Reply by pseudocodes** [reply]
is there any progress ?

**Comment by jose-donato** [comment]
@liihuu do u think it would be possible to build something like this using klinecharts?

**Comment by jose-donato** [comment]
ended up forking klinecharts core and adding a new candle type called `footprint`. still some bugs but just wanted to see if it would be possible. could clean up and create a PR if you think this fits the library @liihuu 

<img width="1450" alt="image" src="https://github.com/user-attachments/assets/d0313bd4-b5a9-48a8-8e30-8f1b876fa055" />

video example [here](https://x.com/josedonato__/status/1937200515110285497)

**Reply by kintailloch** [reply]
Did you create a PR for this or have a repo for the fork?

**Reply by neeerajtk** [reply]
same. @jose-donato pr or any reference of the fork?

**Reply by neeerajtk** [reply]
@jose-donato I guess changes aren't present in your [public fork](https://github.com/jose-donato/KLineChart)?
